packages/p2p/src/stores/general-store.js

```diff
@@ -452,88 +452,90 @@ export default class GeneralStore extends BaseStore {
         );
 
         requestWS({ get_account_status: 1 }).then(({ error, get_account_status }) => {
-            const hasStatuses = statuses => statuses.every(status => get_account_status.status.includes(status));
+            if (!isEmptyObject(get_account_status)) {
```

Comment:
I only wrapped this entire block of code with an if statement to check if get_account_status is not empty